wiki/People/Featured_Artists/en.md

```diff
@@ -41,3 +41,56 @@ All mapping contests published on the [community contest listing](https://osu.pp
 To promote mapping Featured Artist content, members of the community created the [Mappers' Guild](/wiki/Community/Mappers_Guild), giving mappers an MMO-style quest system that they can participate in with other mappers. Prizes are rewarded for creating and ranking beatmaps, including three tiers of profile badges<!-- TODO: profile badges article -->. Mappers can join the guild by visiting the [Mappers' Guild website](https://mappersguild.com/). As of [April 2023](https://osu.ppy.sh/home/news/2023-04-09-new-featured-artist-chon), mappers can also discuss all things related to Featured Artist beatmap creation at the `#featured-artists` channel in the [osu! Discord server](/wiki/Community/osu!_Discord_server).
 
 If a pre-timed `.osz` from the Featured Artist listing has errors of any kind, they can be reported on [this forum thread](https://osu.ppy.sh/community/forums/topics/783985).
+
+## Members
```

Comment:
bit misleading since this list doesn't contain all featured artists, but just the ones that have osu! accounts

```suggestion
## User group
```

very tentative suggestion. if you have a neat idea i'm all ears

Reply:
no, that comes off weird. maybe "Featured Artists with osu! accounts" in that case

Reply:
sure idm going with that

Reply:
```suggestion
## Featured Artists with osu! accounts
```